src/main/frontend/db/react.cljs

```diff
@@ -16,7 +16,7 @@
 ;;; keywords specs for reactive query, used by `react/q` calls
 ;; ::block
 ;; pull-block react-query
-(s/def ::block (s/tuple #(= ::block %) uuid?))
+(s/def ::block (s/tuple #(= ::block %) int?))
```

Comment:
Would be nice to get rid of ::custom at some point as `any?` allows any of this spec to be invalid